server/src/lib/api.ts

```diff
@@ -524,4 +526,21 @@ export default class API {
       await this.model.db.getAccents(client_id, params?.locale || null)
     );
   };
+
+  private getCountQueryParam = (request: Request) => {
```

Comment:
The items that are on the `request.query` object are from the `qs` library and so could be an array or a string. To keep Typescript happy we have to check for that edgecase.

I also noticed we don't check if the number is a number so did that too.